packages/zoe/src/contracts/attestation/attestation.js

```diff
@@ -0,0 +1,158 @@
+// @ts-check
+
+import { AmountMath } from '@agoric/ertp';
+import { E } from '@agoric/eventual-send';
+import { Far } from '@agoric/marshal';
+
+import '../../../exported';
+import { setupAttestation as setupExpiringAttestation } from './expiring/expiringNFT';
+import { setupAttestation as setupReturnableAttestation } from './returnable/returnableNFT';
+import { makeStoredTime } from './storedTime';
+import { max } from './helpers';
+import { assertPrerequisites } from './prerequisites';
+import { makeGetAttMaker } from './attMaker';
+
+/**
+ * @type {ContractStartFn}
+ */
+const start = async zcf => {
+  const {
+    brands: { Underlying: underlyingBrand },
+    authority, // The authority used to confirm that the underlying assets are escrowed appropriately
```

Comment:
Do we want to expose the `authority` in the public terms?  Doing so is not always desirable, especially if direct calls to methods on the authority are not billed.

Reply:
Very good point. I'll remove it from the public terms.